handlers/bind.go

```diff
@@ -22,13 +22,10 @@ func (h APIHandler) Bind(w http.ResponseWriter, req *http.Request) {
 	instanceID := vars["instance_id"]
 	bindingID := vars["binding_id"]
 
-	correlationID := req.Context().Value(middlewares.CorrelationIDKey).(string)
-
 	logger := h.logger.Session(bindLogKey, lager.Data{
-		instanceIDLogKey:             instanceID,
-		bindingIDLogKey:              bindingID,
-		middlewares.CorrelationIDKey: correlationID,
-	})
+		instanceIDLogKey: instanceID,
+		bindingIDLogKey:  bindingID,
+	}, utils.DataForContext(req.Context(), []string{middlewares.CorrelationIDKey}))
```

Comment:
The syntax of passing a `[]string{}` could be improved by turning `DataForContext()` into a variadic function. You would then make calls to `DataForContext(req.Context(), middleware.CorrelationIDKey)` and the function signature would be `DataForContext(context context.Context, dataKeys ...string) lager.Data`. This would have the flexibility of passing more than one key, while looking simpler. What do you think?